.ci/install_cloud_hypervisor.sh

```diff
@@ -15,63 +15,15 @@ readonly script_dir=$(dirname $(readlink -f "$0"))
 cidir=$(dirname "$0")
 arch=$("${cidir}"/kata-arch.sh -d)
 source "${cidir}/lib.sh"
-# Where real kata build script exist, via docker build to avoid install all deps
-latest_build_url="${jenkins_url}/job/cloud-hypervisor-nightly-$(uname -m)/${cached_artifacts_path}"
-clh_bin_name="cloud-hypervisor"
-clh_install_path="/usr/bin/${clh_bin_name}"
-cloud_hypervisor_repo=$(get_version "assets.hypervisor.cloud_hypervisor.url")
-go_cloud_hypervisor_repo=${cloud_hypervisor_repo/https:\/\//}
 
-install_clh() {
-	[ -n "$cloud_hypervisor_repo" ] || die "failed to get cloud_hypervisor repo"
-	export cloud_hypervisor_repo
-
-	# Get version for cloud_hypervisor from runtime/versions.yaml
-	cloud_hypervisor_version=$(get_version "assets.hypervisor.cloud_hypervisor.version")
-	[ -n "$cloud_hypervisor_version" ] || die "failed to get cloud_hypervisor version"
-	export cloud_hypervisor_version
-
-	# Get cloud_hypervisor repo
-	go get -d "${go_cloud_hypervisor_repo}" || true
-	# This may be downloaded before if there was a depends-on in PR, but 'go get' wont make any problem here
+main() {
+	# Just in case the kata-containers repo is not cloned yet.
 	clone_katacontainers_repo
-	pushd  $(dirname "${GOPATH}/src/${go_cloud_hypervisor_repo}")
-	# packaging build script expects run in the hypervisor repo parent directory
-	# It will find the hypervisor repo and checkout to the version exported above
-	"${katacontainers_repo_dir}/tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh"
-	sudo install -D "cloud-hypervisor/${clh_bin_name}"  "${clh_install_path}"
-	popd
-}
-
-install_prebuilt_clh() {
```

Comment:
It is supposed to use a cached clh, otherwise build it. However the clh cache job was disabled a long time ago (apparently around Sept 2020). I just want to double-check if you still want to keep building clh on every job....

Reply:
@wainersm, we usually don't have to, as cloud-hypervisor provides a static binary as part of its release.
We'll have to build it in case we don't use their release, as we do right now.

Do you want to bring back the cloud-hypervisor cache? I wouldn't mind, but that's something to be done out of the scope of this PR.

Reply:
Sorry, I am not used to the clh build process and scripts. Looking at https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh now I realize it gets the built clh or build from source. Great, so the clean up you did make complete sense. Ignore my comment please.